app/views/memberships/edit.html.haml

```diff
@@ -1,53 +1,45 @@
 %h1
-  Edit membership for
+  = t(".edit_membership_for")
   %em
     - if @user.name.present?
       = @user.name
     - else
       = @user.email
 
 %p
-  Joined Bike Index #{@user.created_at.strftime("%-m/%-d/%Y")}
+  - date = @user.created_at.strftime("%-m/%-d/%Y")
+  = t(".joined_bike_index_date", date: date)
 
 %section.full-screen-table.padded
-  %p
-    Bikes indexed for
-    %strong
-      = @membership.organization.name
-    by
-    - if @user.name.present?
-      = @user.name
-    - else
-      = @user.email
+  %p= t(".bikes_indexed_for_org_html", org_name: @membership.organization.name, user: @user.name.presence || @user.email)
+
   %table#org-bikes-table.table.table-striped.table-bordered
     %thead
-      / %th
-      /   Link&nbsp;&nbsp;
       %th
-        Date indexed
+        = t(".date_indexed")
       %th.medium-screens
-        Sent to
+        = t(".sent_to")
       %th
-        Manufacturer
+        = t(".manufacturer")
       %th.large-screens
-        Color&nbsp;&nbsp;
+        = t(".color")
+        &nbsp;&nbsp;
 
     %tbody
       - @bikes.each do |bike|
         %tr
           %td
             - if bike.created_at.today?
-              %span.hidden
-                = bike.created_at.strftime("%-m.%-d.%Y %-H.%M")
-              Today at #{bike.created_at.strftime("%-I %p")}
-            - elsif Date.yesterday().beginning_of_day() <= bike.created_at
-              %span.hidden
-                = bike.created_at.strftime("%-m.%-d.%Y %-H.%M")
-              Yesterday at #{bike.created_at.strftime("%-I %p")}
+              %span.hidden= bike.created_at.strftime("%-m.%-d.%Y %-H.%M")
```

Comment:
1. We have the `convertTime` js stuff now.
2. The `span.hidden` is a remanant of using datatables, so that things were sorted correctly.

So this entire chunk should be removed and replaced with the `convertTime` class and format

Reply:
Also, it might be worth checking to see if this entire table could be wholesale replaced by the organized bikes table or something similar.

Reply:
This template appears to be vestigial -- it's not referenced anywhere and the only memberships controller I'm seeing is admin-namespaced. Removed it in 58a89e9.